spec/dummy/db/schema.rb

```diff
@@ -0,0 +1,25 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 2022_01_12_154220) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "pg_stat_statements"
+  enable_extension "plpgsql"
+
+
+  create_view "pg_stat_statements_info", sql_definition: <<-SQL
```

Comment:
Isn't this the bug we're trying to fix?

Reply:
Oh shit.

Reply:
Yeah that's what happened. Confirmed with `rake dummy:db:migrate` and `rake dummy:db:{drop,create,migrate}`.